testsuite/cluster-test/src/aws.rs

```diff
@@ -4,13 +4,13 @@
 #![forbid(unsafe_code)]
 
 use anyhow::{bail, format_err, Result};
+use libra_logger::*;
```

Comment:
rust style: If I understand correctly, the convention is to typically only include `*` imports from the `prelude` module of crates and use explicit imports for all other modules. Can use `use libra_logger::prelude::*;` here instead?